tensorflow_datasets/core/features/array_2d.py

```diff
@@ -0,0 +1,64 @@
+# coding=utf-8
+# Copyright 2021 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Array2D feature for raising error (Not implemented currently). """
+
+class Array2D:
+    """Array2D feature is used in only two datasets in the community
```

Comment:
As this is a dummy class, let's remove all the docstring to explain instead this is only a dummy wrapper to raise more explicit exception